File: beginner/pagina01/1020/1020.lua
local value = tonumber(io.read())
local years = math.floor(value / 365)
value = value % 365
local months = math.floor(value / 30)
value = value % 30
local days = value


print(string.format("%d ano(s)", years))
print(string.format("%d mes(es)", months))
print(string.format("%d dia(s)", days))

File: beginner/__1000__/1020/1020.py
# ages in days

'''
Leia um valor inteiro correspondente a idade em dias de uma pessoa, e print-a em anos, meses e dias, seguidos respectivamente por “ano(s)”, “mes(es)”, “dia(s)”.
'''

# example 400
# 1 ano(s)
# 1 mes(es)
# 5 dia(s)

age = 400

years = int(age / 365)
mounths = int((age % 365) / 30)
day = int((age % 365) % 30)

print(years, "ano(s)")
print(mounths, "mes(es)")
print(day, "dia(s)")